api/src/main/java/org/openmrs/module/ugandaemrreports/data/converter/IYCFDataConverter.java

```diff
@@ -25,16 +25,15 @@
     public Object convert(Object obj) {
 
         if (obj == null) {
-            return "";
+            return null;
         }
         Concept iyfcResults = ((Obs) obj).getValueCoded();
```

Comment:
@solemabrothers see suggested changes on the NationalityPersonalAttributeDataConverter if checks and returns

Reply:
Done